ts/components/SendSupportRequestOptions.tsx

```diff
@@ -7,30 +7,107 @@ import themeVariables from "../theme/variables";
 import ButtonDefaultOpacity from "./ButtonDefaultOpacity";
 import { H1 } from "./core/typography/H1";
 import { BaseHeader } from "./screens/BaseHeader";
-import Accordion from "./ui/Accordion";
 import Markdown from "./ui/Markdown";
 import IconFont from "./ui/IconFont";
 import FooterWithButtons from "./ui/FooterWithButtons";
-import { Label } from "./core/typography/Label";
-import { RawCheckBox } from "./core/selection/RawCheckBox";
 import { EdgeBorderComponent } from "./screens/EdgeBorderComponent";
 import { IOStyles } from "./core/variables/IOStyles";
+import { RawCheckBox } from "./core/selection/RawCheckBox";
+import { Label } from "./core/typography/Label";
+import Accordion from "./ui/Accordion";
+
+const checkboxStyle = StyleSheet.create({
+  container: {
+    flexDirection: "row",
+    alignItems: "flex-start",
+    justifyContent: "space-evenly"
+  }
+});
+
+/**
+ * Checkboxes we need as of now
+ */
+
+export enum CheckboxIDs {
+  sendScreenshot = "sendScreenshot",
+  sendPersonalInfo = "sendPersonalInfo"
+}
+
+/**
+ * Checkbox+Label+Optional Accordion component
+ */
+
+type CheckboxFormItemProps = {
+  target: CheckboxIDs;
+  isChecked?: boolean;
+  onToggle: () => void;
+};
+
+type CheckboxLabelMap = {
+  [key in CheckboxIDs]: {
+    cta: string;
+    accordion?: {
+      title: string;
+      content: string;
+    };
+  };
+};
+
+const CheckboxLabelKeys: CheckboxLabelMap = {
+  sendScreenshot: {
+    cta: I18n.t("contextualHelp.sendScreenshot.cta")
+  },
+  sendPersonalInfo: {
+    cta: I18n.t("contextualHelp.sendPersonalInfo.cta"),
+    accordion: {
+      title: I18n.t("contextualHelp.sendPersonalInfo.informativeTitle"),
+      content: I18n.t("contextualHelp.sendPersonalInfo.informativeDescription")
+    }
+  }
+};
+
+export const CheckBoxFormItem: React.FC<CheckboxFormItemProps> = ({
```

Comment:
Since this is not a common CheckBox component but it is specific for ContextualHelp, maybe a good idea can be naming it properly, what do you think?

In addition, add some comments to specify what is this component doing